SourceCpp/Params/param_includes/pelec_defaults.H

```diff
@@ -3,17 +3,17 @@
 // or add runtime parameters, please edit _cpp_parameters and then run
 // mk_params.sh
```

Comment:
There shouldn't be any chances here beyond the soot stuff. And this file should be auto generated through `mk_params.sh`